src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/entity/MixinEntityPlayerSP.java

```diff
@@ -333,6 +333,14 @@ public void onLivingUpdate() {
             sprintToggleTimer = 0;
         }
 
+        if (isSneaking()) {
+            final SneakSlowDownEvent sneakSlowDownEvent = new SneakSlowDownEvent(0.2F, 0.2F);
+            EventManager.INSTANCE.callEvent(sneakSlowDownEvent);
+            movementInput.moveStrafe *= (sneakSlowDownEvent.getStrafe() * 5);
+            movementInput.moveForward *= (sneakSlowDownEvent.getForward() * 5);
+            // Uncommenting this until I figure out what this is sprintToggleTimer = 0;
+        }
```

Comment:
The event takes 0.2f so you multiply it by * 5, so it turns to 1 and mulitply moveStrafe and moveForward by 1, which does (nothing). This looks way to complicated, might just get rid of the 0.2F and turn it into 1 instead of adding useless math.